src/pages/invoices/email/components/Mailer.tsx

```diff
@@ -40,14 +40,15 @@ export type MailerResource = Invoice | RecurringInvoice | Quote | PurchaseOrder;
 
 export type MailerContactProperty = 'client_id' | 'vendor_id';
 interface Props {
+  ref: RefObject<HTMLInputElement | undefined>;
   resource: MailerResource;
   resourceType: MailerResourceType;
   list: Record<string, string>;
   defaultEmail: string;
   redirectUrl: string;
 }
 
-export function Mailer(props: Props) {
+export const Mailer = forwardRef<MailerComponent, Props>((props, ref) => {
```

Comment:
I haven't found a way to implement it with function based components. All the examples I've found are with arrow-based components.